### PR TITLE
tests: increase timeout in rolling update tests

### PR DESCRIPTION
**Pull request by justinsb**
We never know when e.g. a GC is going to delay us, so we need a lot
more padding on these timeouts.

### CODE REVIEW

File: pkg/instancegroups/rollingupdate_test.go [hunk 1]
```diff
@@ -538,7 +538,11 @@ func (v *flappingClusterValidator) Validate() (*validation.ValidationCluster, er
 func TestRollingUpdateFlappingValidation(t *testing.T) {
 	c, cloud, cluster := getTestSetup()
 
-	c.ValidationTimeout = 20 * time.Millisecond
+	// This should only take a few milliseconds,
+	// but we have to pad to allow for random delays (e.g. GC)
+	// TODO: Replace with a virtual clock?
+	c.ValidationTimeout = 200 * time.Second
```
Comment (johngmyers) [inline pkg/instancegroups/rollingupdate_test.go]: 200 seconds is a  lot of time. I'm sure 1 second would be more than sufficient.

Comment (justinsb) [reply pkg/instancegroups/rollingupdate_test.go]: Oops... guess who switched from "200ms is enough" to thinking "hmm, that looks oddly specify; better just to call it one second" :-)

**Comment by johngmyers** [comment]
LGTM FWIW

**Pull request merged**